pg_backup_api/pg_backup_api/spec/pg_backup_api.yaml

```diff
@@ -9,13 +9,861 @@ components:
       required:
```

Comment:
(This comment is actually for line 4 but GitHub won't let me add a comment that far away from the diff, apparently.) 

This is probably ready enough to set the API version to 1.0, or at least 0.1 (it is currently on 0.0.1).